Formula/aris.rb

```diff
@@ -0,0 +1,30 @@
+class Aris < Formula
+  desc "Logical proof program"
+  homepage "https://github.com/kovzol/aris"
+  url "https://github.com/kovzol/aris/archive/refs/tags/2.2-15042023.tar.gz"
+  sha256 "7c44f0d53c1c2ab535653159f9aac47c9614426756d6f7c35bea19024331d3b1"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "cairo"
+  depends_on "coreutils"
+  depends_on "gtk+3"
+  depends_on "pango"
+  depends_on "pkg-config"
+  uses_from_macos "libxml2"
+
+  on_linux do
+    depends_on "libxml2"
+  end
```

Comment:
These two things are identical. If it uses the system libxml2 on macOS that means it depends on it on linux.
```suggestion
  uses_from_macos "libxml2"
```

Reply:
Thanks, I fixed this.

Reply:
So --- is there a chance to get this into homebrew, or do I need to create my tap instead?